4] guessing game 3/main.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template, request
+
+app = Flask(__name__)
+
+min_value = 0
+max_value = 1000
+guess = 500
+
+@app.route("/", methods=["GET", "POST"])
+def index():
+    global min_value, max_value, guess
```

Comment:
globalni prommene nejsou dobra praxe. To se neuc pouzivat. To je hack. Neni to potreba, nainicializuj si je normalne uvnitr